docs/src/developer/standards.rst

```diff
@@ -1,47 +1,86 @@
 
-Coding and Repository Standards
-===============================
-
-We are using flake8_ (along with some of its plugins) and pylint_.
-Their styles are provided in ``/tox.ini`` and ``/.pylintrc`` respectively.
+You can verify if your code will pass the checks locally beforehand using:
 
 .. code-block:: sh
 
-   tox -e flake8
-   tox -e pylint
+   tox -e lint
 
-Also, we are using a check-manifest_ which compares ``/MANIFEST.in`` and git
-structure of the source repository, and finally readme_renderer_ which
-checks whether ``/README.rst`` can be
-actually rendered in PyPI_ website page for Oríon.
+.. _standard-coding:
 
-.. code-block:: sh
+Coding standard
+===============
 
-   tox -e packaging
+Our coding standards are specified via flake8_ and pylint_. Their configurations are provided in
+``tox.ini`` and ``.pylintrc`` respectively. You can verify the conformity of your changes locally
+by running ``$ tox -e flake8`` and ``$ tox -e pylint``.
 
-To run all of expected linters execute::
+.. _standard-vcs:
 
-   tox -e lint
+Version Control Guidelines
+==========================
 
-.. _flake8: http://flake8.pycqa.org/en/latest/
-.. _pylint: https://www.pylint.org/
-.. _check-manifest: https://pypi.org/project/check-manifest/
-.. _readme_renderer: https://pypi.org/project/readme_renderer/
-.. _PyPI: https://pypi.org/
+To collaborate through VCS, we follow the
+`gitflow <https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow>`_
+workflow. The *develop* and *master* branches are protected and can only be changed with pull
+requests.
+
+For branch names, we recommend prefixing the name of the branch with *feature/*, *fix/*, et
+*doc/* depending on the change you're implementing. Additionally, we encourage adding the issue's id
+at the start of the branch name, after the prefix. For example the branch name for a bug represented
+by issue 225 would be ``fix/225-short-bug-description``.
+
+When creating a release, we use the pattern *release-{version}rc*. This branch represent the release
+candidate that will be merge in the master branch when the changes are ready to be launched in
+production.
+
+Synchronization
+---------------
+Regarding merges, we recommend you keep your changes in your forked repository for as long as
+possible and rebase your branch to Oríon's develop branch before submitting a PR.
+Most probably, the develop branch will have changed by the time your PR is approved. In such cases,
+we recommend to merge the changes from develop to your branch and then merge your branch to develop.
+We discourage rebases after the PR has been submitted as it can cause problems in GitHub's review
+system. On another note, merges are always done with the creation of a merge commit, also known
+as a *non fast-forward merge*.
 
-Fork and Pull Request
-=====================
+In some cases where the PR has small and focused changes contained in one or two commits,
```

Comment:
I would say it's more about contributions which are scattered across many commits with many incremental commits such as `fix pep8` or `update based on feedback`.